ext/DFTKAMDGPUExt.jl

```diff
@@ -12,4 +13,23 @@ function LinearAlgebra.cholesky(A::Hermitian{T, <:AMDGPU.ROCArray}) where {T}
     LinearAlgebra.Cholesky(Acopy, A.uplo, info)
 end
 
```

Comment:
Do you not need some fencing like https://github.com/JuliaMolSim/DFTK.jl/blob/master/ext/DFTKCUDAExt.jl#L20 here ?